src/PhpSpec/Locator/ResourceManager.php

```diff
@@ -18,14 +18,14 @@
     /**
      * @param string $query
      *
-     * @return Resource[]
+     * @return \PhpSpec\Locator\Resource[]
```

Comment:
not needed as the class is in the current namespace and so the short name works already